app/view_models/profile_index.rb

```diff
@@ -16,13 +16,21 @@ def personal_key_partial
   end
 
   def password_reset_partial
-    if current_user.password_reset_profile.present?
+    if current_user.decorate.password_reset_profile.present?
```

Comment:
Does it need to be public in the controller? Or do we pass in a `DecoratedUser` to the `ProfileIndex` view model? Both `current_user` and `decorated_user` get used in the view model, so I don't know that it really saves any performance or readability either way.